views/world.pug

```diff
@@ -20,6 +20,12 @@ block content
         .row
             dt Description: 
             dd #{description}
+        .row
+            dt First published on: 
+            dd #{firstPublishTime}
```

Comment:
```suggestion
            dd #{firstPublishTime} UTC
```

Reply:
Would replacing the `GMT` with `UTC` in the string suffice? I fear that having both after one another is too redundant.

![zen_93JonsY0Bk](https://github.com/user-attachments/assets/18867728-ac45-4884-8a96-335edfe82143)

Reply:
OH IM SORRY, does the string already contain UTC?

Reply:
Oh its GMT that's fine

I'm tired sorry!

Reply:
Well instead it contains `GMT` which might not be as explicit to users while being the same as `UTC` in practice.

Reply:
Its all good. I just had to go look at the JavaScript specification for a bit and scratch my head.